src/lib/OpenEXRUtil/ImfCheckFile.cpp

```diff
@@ -1453,6 +1453,23 @@ bool checkCoreFile(exr_context_t f, bool reduceMemory, bool reduceTime)
 
 ////////////////////////////////////////
 
+static void
+core_error_handler_cb (exr_const_context_t f, int code, const char* msg)
+{
+#if 0
```

Comment:
Dead code?

Reply:
well, yes and no. I should add a comment to this at the very least. Peter requested to silence the error prints to make the oss fuzz logs smaller and easier to parse, but then I also want the debug prints to know something has happened. Let me circle back with him and we can decide how to enable it. perhaps an env var instead of if 0 and re-compile.